Swiftfin/Views/SettingsView/UserDashboardView/DevicesView/Components/DeviceRow.swift

```diff
@@ -0,0 +1,198 @@
+
+        // MARK: - Row Content
+
+        @ViewBuilder
+        private var rowContent: some View {
+            HStack {
+                VStack(alignment: .leading, spacing: 5) {
+                    Text(deviceInfo.name ?? L10n.unknown)
+                        .font(.title3)
+                        .fontWeight(.semibold)
+                        .lineLimit(2)
+                        .multilineTextAlignment(.leading)
+
+                    Text(deviceInfo.lastUserName ?? L10n.never)
+                        .fontWeight(.semibold)
+                        .lineLimit(1)
+                        .multilineTextAlignment(.leading)
+
+                    TextPairView(
+                        deviceInfo.appName ?? L10n.unknown,
+                        value: Text(deviceInfo.appVersion ?? .emptyDash)
+                    )
+
+                    TextPairView(
+                        L10n.lastSeen,
+                        value: {
+                            if let dateLastActivity = deviceInfo.dateLastActivity {
+                                Text(dateLastActivity, format: .relative(presentation: .numeric, unitsStyle: .narrow))
+                            } else {
+                                Text(L10n.never)
+                            }
+                        }()
+                    )
+                    .id(currentDate)
+                    .monospacedDigit()
+                }
+                .foregroundStyle(labelForegroundStyle, .secondary)
+
+                Spacer()
+
+                if isEditing, isSelected {
+                    Image(systemName: "checkmark.circle.fill")
+                        .resizable()
+                        .backport
+                        .fontWeight(.bold)
+                        .aspectRatio(1, contentMode: .fit)
+                        .frame(width: 24, height: 24)
+                        .symbolRenderingMode(.palette)
+                        .foregroundStyle(accentColor.overlayColor, accentColor)
+
+                } else if isEditing {
+                    Image(systemName: "circle")
+                        .resizable()
+                        .backport
+                        .fontWeight(.bold)
+                        .aspectRatio(1, contentMode: .fit)
+                        .frame(width: 24, height: 24)
+                        .foregroundStyle(.secondary)
+                }
+            }
+        }
+
+        // MARK: - Body
+
+        var body: some View {
+            ListRow(insets: .init(vertical: 8, horizontal: 8)) {
+                deviceImage
+                    .padding(.trailing, 8)
+            } content: {
+                rowContent
+            }
+            .onSelect(perform: onSelect)
+            .swipeActions {
+                Button {
+                    onDelete()
+                } label: {
+                    Label(L10n.delete, systemImage: "trash")
+                }
+                .tint(.red)
+            }
+            .contextMenu {
```

Comment:
We don't need both swipe action and context menu, just have the swipe action delete.